google/cloud/spanner/client.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/spanner/client.h"
+#include "google/cloud/spanner/value.h"
+#include "google/cloud/grpc_utils/grpc_error_delegate.h"
+#include "google/cloud/log.h"
+
+namespace google {
+namespace cloud {
+namespace spanner {
+inline namespace SPANNER_CLIENT_NS {
+
+namespace spanner_proto = ::google::spanner::v1;
+
+StatusOr<Client> Client::MakeClient(ClientOptions const& client_options) {
+  return Client(internal::CreateDefaultSpannerStub(client_options));
+}
+
+StatusOr<ResultSet> Client::Read(std::string const& /*table*/,
+                                 KeySet const& /*keys*/,
+                                 std::vector<std::string> const& /*columns*/,
+                                 ReadOptions const& /*read_options*/) {
+  return Status(StatusCode::kUnimplemented, "not implemented");
```

Comment:
I believe this is a 1-liner forwarding to the overload that takes a `Transaction::SingleUseOptions`.